sale_purchase_order_line_report/__manifest__.py

```diff
@@ -5,8 +5,12 @@
     "summary": """
         Create a view that combines purchase and sales order lines,
         accessible from both partner and product.
+        There are two modes of viewing the view:
+        - report on ordered quantities
+        - report on received/delivered quantities
+        configurable in the settings.
     """,
-    "version": "17.0.1.0.1",
+    "version": "17.0.1.1.0",
```

Comment:
Better switching to major version, as this is a huge change:
```suggestion
    "version": "17.0.2.0.0",
```